### DISCUSSION TITLE
Override Admin Login Template

### DISCUSSION THREAD

**Discussion by polymed** [pimcore/pimcore#8792] [Q&A / Support]
Hi guys,

can someone tell me how to override the admin login template? It doesn't seem to work the way it is described in the Symfony docs. https://symfony.com/doc/current/bundles/override.html

Pimcore-Version: 6.8.12

Best regards

**Comment by brusch** [comment]
@polymed Should work, however, please be aware that the copyright has to stay in place! 
Probably better to use the offered branding settings 😊

**Reply by passioneight** [reply]
@brusch an example use case for overriding is adding SSO-buttons to the login page.

**Reply by polymed** [reply]
@brusch I think I've tried every variation now, but nothing works.

```
templates
└── bundles
    ├── AdminBundle
    │   └── Admin
    │       └── Login
    │           └── login.html.php
    └── PimcoreAdminBundle
        └── Admin
            └── Login
                └── login.html.php
```

**Reply by passioneight** [reply]
Try the approach explained [in this discussion](https://github.com/pimcore/pimcore/discussions/9204).

**Comment by mjnewman** [comment]
@brusch when we set the Custom Logo and Custom Login Background Image within the System Settings, it saves the settings and reloads the admin interface. We then see the logo and image as expected. However, after logging out, the logo and image revert to the Pimcore defaults.

Are there any additional steps we need to take to get the custom branding to remain in place? We're using Pimcore v6.8.10.

Thanks!

**Reply by passioneight** [reply]
Have you tried clearing the cache?
Try executing `bin/console cache:clear` and `bin/console pimcore:cache:clear` - also clear your browser cache.

**Reply by mjnewman** [reply]
Thanks – Will definitely try clearing the Pimcore cache.

It does appear properly at first, but after a few logins it reverts back to the default. No one else has access to change the settings.
